### PR TITLE
Course schedule oauth

### PR DESCRIPTION
**Pull request by mxcosu**

### CODE REVIEW

File: src/features/ScheduleCard.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  );
+
+  // const selectedPlannerItems = [];
+  // Get a list of days with courses or assignments.
+  // Used to display the orange dots above days to indicate
+  // which days have events at a quick glance.
+  const daysWithEvents = useMemo(
+    () =>
+      nextFiveDays.filter(day => {
+        let dayShortcode = getDayShortcode(day);
```
Comment (revgum) [inline src/features/ScheduleCard.tsx]: should this be `const`?

File: src/features/ScheduleCard.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  // Populate assignment data for current user
+  useEffect(() => {
+    getAcademicCalendarEvents()
+      .then(setCalEvents)
+      .catch(console.log);
+  }, []);
+
+  const getCoursesOnSelectedDay = () => {
+    let selectedDayShortcode = getDayShortcode(selectedDay);
```
Comment (revgum) [inline src/features/ScheduleCard.tsx]: should this be `const`?

File: src/features/ScheduleCard.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    );
+  } else {
+    selectedPlannerItems = [];
+  }
+
+  let selectedCalEvents = calEvents.filter(event =>
+    event.pubDate ? isSameDay(event.pubDate, selectedDay) : ''
+  );
+
+  // const selectedPlannerItems = [];
```
Comment (revgum) [inline src/features/ScheduleCard.tsx]: should this be removed?

**Comment by mxcosu** [comment]
Thanks all updated.

**Pull request merged**